docs/ch_blocks.adoc

```diff
@@ -20,22 +20,9 @@ message asking you to update the vectors.  Once these two vectors have
 been updated, you can use the block words.
 
 If you would like to play with some blocks, but don't have any
-hardware or are running Tali in a simulator, fear not!  Tali has a
-built-in RAM drive that can be accessed by running:
+hardware or are running Tali in a simulator, fear not! Tali has a
+built-in RAM drive that can be used on any hardware or simulator.  If using the
+c65 simulator that comes with Tali, persistant storage can be simulated using
```

Comment:
persistent

Reply:
Thanks - flag any of those that you find (I'm sure there are more in there).  I re-read everything when I'm done, but don't always catch them all.